src/DataMapper/Product/Provider/ProductPricesProvider.php

```diff
@@ -8,20 +8,14 @@
 use Sylius\Component\Core\Model\ChannelInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariantInterface;
-use Sylius\Component\Product\Resolver\ProductVariantResolverInterface;
 
 final class ProductPricesProvider implements ProductPricesProviderInterface
 {
-    public function __construct(
```

Comment:
This shouldn't be removed because this is the default service for resolving a variant in Sylius

Reply:
But it causes issues on fixtures load. Additionally, the logic I used is almost the same as what this service does (the except is not using the repository).

Reply:
I created this PR: https://github.com/Setono/sylius-meilisearch-plugin/pull/18 specifically for this bug :)

Reply:
Yup, reverted locally.